From 99b5bbe756a74ecec4ca728066fee7373a477b9e Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 21 Sep 2018 10:35:07 -0600
Subject: [PATCH 4/6] klte-common: Stop building sensors.msm8974

* msm8974-common has moved to hwbinder mode for sensors, with the
  multihal statically linked into the impl. We don't need this
  version of the multihal anymore.

Change-Id: Id9e9103bb03351cef2a7cb0e81800057898fe862
---
 klte.mk | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/klte.mk b/klte.mk
index 5103a23..62eb011 100644
--- a/klte.mk
+++ b/klte.mk
@@ -90,9 +90,6 @@ PRODUCT_PACKAGES += \
     ueventd.qcom.rc
 
 # Sensors
-PRODUCT_PACKAGES += \
-    sensors.msm8974
-
 PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/configs/sensors/_hals.conf:$(TARGET_COPY_OUT_VENDOR)/etc/sensors/_hals.conf
 
-- 
2.17.1

